VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/interactor/LandingScreenInteractor.java

```diff
@@ -21,6 +21,11 @@ public LandingScreenInteractor observeLandingScreen() {
         return this;
     }
 
+    public RegistrationScreenInteractor navigateToRegistrationScreen() {
```

Comment:
Should 'registration' be replaced with 'signUp' in all usages here, since that's the existing terminology in the existing `onSignUpView()` method, and the title of the screen as well?